sort.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+function bubbleSort($sort_array)
+{
+    if (empty($sort_array)) {
+        return [];
+    }
+
+    $array_indexes = count($sort_array);
+    $sorted = false;
+
+    for ($i = 0; $i < $array_indexes; $i++) {
+        for ($j = 0; $j < $array_indexes - $i - 1; $j++) {
+            if ($sort_array[$j + 1] < $sort_array[$j]) {
+                $tmp = $sort_array[$j + 1];
+                $sort_array[$j + 1] = $sort_array[$j];
+                $sort_array[$j] = $tmp;
+
+                $sorted = true;
+            }
+        }
+
+        if (! $sorted) {
+            break;
+        }
+
+        if (strpos($sort_array[$i], "\n") === false) {
+            $sort_array[$i] .= "\n";
+        }
```

Comment:
Please put this out of the bubbleSort function (and do it before sorting) as this has nothing to do with bubble sort.

Reply:
Where should I implement this function before or after bubblesort function?

Reply:
As I said: before.